### PR TITLE
 English homework has been done. Lecture 1 

### PR DESCRIPTION
**Pull request by dramlin**

### CODE REVIEW

**Review by VasiliZas — APPROVED** [review]

**Review by vitya771 — APPROVED** [review]

**Review by gornovanton1992 — APPROVED** [review]
lgfm

File: homework/english/Petrov_Denis/L1/HomeworkLecture1 [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+7 to retool  - a system administrator has to retool its workplace.
+8 to implement - a class implements a interface.
+9 to run a program - You should run Firefox to browse the Internet
+10 to enable - Acronis enables to backup a data.
+11 to utilize - Programmers utilizes a various api of services.
+12 language features - trait is  language features of PHP.
+13 multithreaded - Java is a multithreaded programming language.
+14 electronic appliance - electronic appliances is used everywhere.
+15 embedded systems  - embedded systems based on a Raspberry Pi.
+16 to complete a course  - I will try to complete a course.
```
Comment (Uchukaknado) [inline homework/english/Petrov_Denis/L1/HomeworkLecture1:125]: I will try to complete the course.

File: homework/english/Petrov_Denis/L1/HomeworkLecture1 [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+6 to support - Microsoft doesn't support Windows XP.
+7 to retool  - a system administrator has to retool its workplace.
+8 to implement - a class implements a interface.
+9 to run a program - You should run Firefox to browse the Internet
+10 to enable - Acronis enables to backup a data.
+11 to utilize - Programmers utilizes a various api of services.
+12 language features - trait is  language features of PHP.
+13 multithreaded - Java is a multithreaded programming language.
+14 electronic appliance - electronic appliances is used everywhere.
+15 embedded systems  - embedded systems based on a Raspberry Pi.
```
Comment (Uchukaknado) [inline homework/english/Petrov_Denis/L1/HomeworkLecture1:124]: Embedded systems are based on a Raspberry Pi.

File: homework/english/Petrov_Denis/L1/HomeworkLecture1 [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+2 to transfer - a data is transfered via the Internet.
+3 to compile - a source code is compiled to a byte code.
+4 to pass to - a link to pass to a other page.
+5 to switch - a trigger are used to switch a state.
+6 to support - Microsoft doesn't support Windows XP.
+7 to retool  - a system administrator has to retool its workplace.
+8 to implement - a class implements a interface.
+9 to run a program - You should run Firefox to browse the Internet
+10 to enable - Acronis enables to backup a data.
+11 to utilize - Programmers utilizes a various api of services.
```
Comment (Uchukaknado) [inline homework/english/Petrov_Denis/L1/HomeworkLecture1:120]: Programmers utilize various api services.

File: homework/english/Petrov_Denis/L1/HomeworkLecture1 [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+23 to contain - A virtual mashine can contain other virtual mashines.
+24 placeholder - To continue you should fill placeholders
+25 lowercase letter - :
+26 capital letter - the editor allows you to enter lowercase letter and capital letter
+27 purpose -   the purpose of an Information Technology is to help people carry out their work easier.
+28 variable - a integer varibale should not contain a real value. 
+29 discouraged - it is discouraged to use depresated functional.
+30 to avoid -  in some cases, you should avoid using recursion.  
+32 analytics engine - a analytics engine use deep learning.
+33 distibuted search  - distibuted search speed up searching for required infotamtion
```
Comment (Uchukaknado) [inline homework/english/Petrov_Denis/L1/HomeworkLecture1:141]: distibuted search 

- [ ] speeds up searching for required infotamtion

**Review by Uchukaknado — COMMENTED** [review]
Well done! Check some corrections.

**Pull request merged**